py-substrate-interface/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Test and Deploy
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
```

Comment:
Can we use one python version for all the projects?

Reply:
Potentially, but it should be in a sepaterate PR I think.